Ansible/roles/liferay-csgf/tasks/domain.yml

```diff
@@ -133,7 +133,7 @@
   command:
```

Comment:
If you have taken out the condition, won't this get triggered always ? Should there be a different codition, or should it remain as this commit ?